src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts\Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  display: block;
+  font-weight: 300;
+}
+
+.form {
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.form-1 {
+  height: 68px;
+}
+
+.form-2 {
+  height: 40px;
+}
+
+.big__img {
+  margin-right: 17px;
+  margin-left: 25px;
+  margin-top: 24px;
+}
+
+.big__search {
+  border: none;
+  display: block;
+  position: absolute;
+  top: 43px;
+  left: 70px;
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.big__search::placeholder {
+  color: #3d4e61;
+}
+
+.big__search:hover {
+  border: 1px solid #3d4e61;
+}
+
+.big__search:focus {
+  font-family: Avenir-Heavy, Arial, Helvetica, sans-serif;
+}
```

Comment:
You have same styles here for big__search and small__search.
Create and apply common class.